doc/mgmt/Management Framework.md

```diff
@@ -841,37 +1093,80 @@ Translib is a library that adapts management server requests to SONiC data provi
         func Replace(req SetRequest) (SetResponse, error)
         func Delete(req SetRequest) (SetResponse, error)
         func Get(req GetRequest) (GetResponse, error)
-        func Subscribe(paths []string, q *queue.PriorityQueue, stop chan struct{}) ([]*IsSubscribeResponse, error)
-        func IsSubscribeSupported(paths []string) ([]*IsSubscribeResponse, error)
+        func Action(req ActionRequest) (ActionResponse, error)
+        func Bulk(req BulkRequest) (BulkResponse, error)
```

Comment:
Do you have REST performance numbers to share to community? ex: add-delete-add vlan workflows etc..